src/custom/state/claim/hooks/index.ts

```diff
@@ -1,25 +1,33 @@
-import { useMemo } from 'react'
+import { useCallback, useMemo } from 'react'
 import JSBI from 'jsbi'
 import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { TransactionResponse } from '@ethersproject/providers'
 
-import { useUserClaims } from 'state/claim/hooks/hooksMod'
 import { useVCowContract } from 'hooks/useContract'
-import { useSingleContractMultipleData } from 'state/multicall/hooks'
 import { useActiveWeb3React } from 'hooks/web3'
+import { useSingleContractMultipleData } from 'state/multicall/hooks'
+import { useTransactionAdder } from 'state/enhancedTransactions/hooks'
+
 import { V_COW } from 'constants/tokens'
 
+import { formatSmart } from 'utils/format'
+import { calculateGasMargin } from 'utils/calculateGasMargin'
+
+import { useUserClaims } from 'state/claim/hooks/hooksMod'
+
 export * from './hooksMod'
 
-export type ClaimType =
-  | 'Airdrop' // free, no vesting, can be available on both mainnet and gchain
-  | 'Team' // free, with vesting, only on mainnet
-  | 'Advisor' // free, with vesting, only on mainnet
-  | 'GnoOption' // paid, with vesting, must use GNO, can be available on both mainnet and gchain
-  | 'UserOption' // paid, with vesting, must use Native currency, can be available on both mainnet and gchain
-  | 'Investor' // paid, with vesting, must use USDC, only on mainnet
+export enum ClaimType {
```

Comment:
haaaa yes enums lol. export const enum? wasn't that smaller or sth

Reply:
Just to be inline with the contract, which uses enums, which are translated as 0-indexed integers and I would have to have a map if using types :shrug: 

Apparently there's no definition or something https://thisthat.dev/const-enum-vs-enum/
Will update